net-proxy/haproxy/haproxy-1.6.9.ebuild

```diff
@@ -119,7 +123,6 @@ src_install() {
 	newconfd "${FILESDIR}/${PN}.confd" $PN
 	newinitd "${FILESDIR}/${PN}.initd-r3" $PN
 
-	dodoc CHANGELOG CONTRIBUTING MAINTAINERS
```

Comment:
There is no ``default`` or ``einstalldocs``. So I would guess that with this change docs will no longer be installed.